pygmt/datasets/map_tiles.py

```diff
@@ -0,0 +1,108 @@
+"""
+Function to load raster basemap tiles from XYZ tile providers, and load as
+:class:`xarray.DataArray`.
+"""
+
+try:
+    import contextily
+except ImportError:
+    contextily = None
+
+import numpy as np
+import xarray as xr
+
+
+def load_map_tiles(region, source=None, lonlat=False, **kwargs):
+    """
+    Load a georeferenced raster basemap from XYZ tile providers.
+
+    The tiles that compose the map are merged and georeferenced into an
+    :class:`xarray.DataArray` image with 3 bands (RGB). Note that the returned
+    image is in a Spherical Mercator (EPSG:3857) coordinate reference system.
+
+    Parameters
+    ----------
+    region : list
+        The bounding box of the map in the form of a list [*xmin*, *xmax*,
+        *ymin*, *ymax*]. These coordinates should be in Spherical Mercator
+        (EPSG:3857) if ``lonlat=False``, or longitude/latitude if
+        ``lonlat=True``.
```

Comment:
I think we should set `lonlat` to `True` by default.

Reply:
Yeah, was thinking about this too since  lonlat bounding box coordinates is more common. The default for `contextily.bounds2img` is `ll=False` though 🙃

Reply:
Ok, changed default to `lonlat=True` in f913222ff1bf480a1dbc4873eb2ada48c968a1c5.